airbyte-integrations/connectors/source-sendgrid/sample_files/catalog.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
is this file the same as the one in `integration_test`? if so, let's get rid of one of them. maybe the one in integration_test can be removed? if you do that, you'll need to adjust the `dockerfile.test` to copy the file from `sample_files` so that the standard tests will have access to the file. let me know if that isn't clear!